packages/blockchain-extension/extension/commands/submitTransaction.ts

```diff
@@ -332,6 +333,22 @@ export async function submitTransaction(evaluate: boolean, treeItem?: Instantiat
 
             outputAdapter.show(); // Bring the 'Blockchain' output channel into focus.
 
+
+            const extensionData: ExtensionData = GlobalState.get();
+
+            // On the user's first transaction submit, show a link to a survey
+            if (!evaluate && !extensionData.shownFirstSubmissionSurveyURL) {
+                // Show review box when we know that it has been successful
+                const surveyMessage: string = 'Congratulations on submitting your first transaction! We appreciate your feedback to help improve the extension';
+                const surveyCallToAction: string = 'Open survey';
+                const surveyURL: string = 'https://www.surveygizmo.com/s3/5561397/VSCodeExt-Popup-Link';
+                await GlobalState.update({ ...extensionData, shownFirstSubmissionSurveyURL: true });
+
+                // Don't use async/await here so that the parent function can return
```

Comment:
Can you add `// ts-ignore` above this to get the warning to go away?

Reply:
Have added `// tslint:disable-next-line: no-floating-promises`